kk/models/images.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.conf import settings
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+from django.core.files.storage import FileSystemStorage
+
+from .base import ModifiableModel
+from .hearing import Hearing
+
+def get_images_dir():
+    return '%s/%s/' % (settings.BASE_DIR, settings.IMAGES_DIR)
+
+file_storage = FileSystemStorage(location=get_images_dir())
+
+class CommonImage(ModifiableModel):
+    IMAGE_TYPE_ORIGINAL = '1'
```

Comment:
What are the semantics of these image types? Can't we just use `easy_thumbnails` or similar to deal with generating the derivatives?